proxy/http/HttpTransact.h

```diff
@@ -905,8 +905,9 @@ class HttpTransact
     int next_hop_scheme; // out
     int orig_scheme;     // pre-mapped scheme
     int method;
-    int cause_of_death_errno; // in
-    HostDBInfo host_db_info;  // in
+    int cause_of_death_errno;     // in
+    Ptr<HostDBInfo> hostdb_entry; // Pointer to the entry we are referencing in hostdb-- to keep our ref
```

Comment:
If it is not used why do you need to hold a refcount? ``host_db_info`` is a copy IIRC. I didn't see anything that needed ``hostdb_entry``.